osu.Game/Online/API/Requests/Responses/APIRelation.cs

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using Newtonsoft.Json;
+
+namespace osu.Game.Online.API.Requests.Responses
+{
+    public class APIRelation
+    {
+        [JsonProperty("target_id")]
+        public int TargetID { get; set; }
+
+        [JsonProperty("relation_type")]
+        public RelationType RelationType { get; set; }
```

Comment:
Shouldn't either this field, or the enum type itself, be annotated with `[StringEnumConverter]` or something?

Reply:
sorry, I forgot it.
Thanks for pointing out